dev/tilt.bzl

```diff
@@ -50,15 +18,22 @@ def _invoke_tilt(ctx: AnalysisContext, subcmd: str) -> list[[DefaultInfo, RunInf
         ctx.attrs.tiltfile,
     )
 
+    script = ctx.actions.write("tilt-run.sh", """\
+#!/usr/bin/env bash
+set -euo pipefail
+
+rootpath="$(git rev-parse --show-toplevel)"
```

Comment:
Not sure if git here is an acceptable dependency.

This approach was derived from si after I'd exhausted all ways to get a handle on the root dir from the bzl file otherwise without having to execute from python

Reply:
We're bringing in `gitMinimal` so this should be ok